Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm

```diff
@@ -406,6 +406,13 @@ - (void)_dismissContactPickerWithContacts:(NSArray *)contacts
 #endif
 }
 
+- (void)_simulateSelectionStart
+{
+#if PLATFORM(IOS_FAMILY)
```

Comment:
NIT: Could we put this in `WKWebViewTestingIOS.mm` (and `WKWebViewPrivateForTestingIOS.h`) instead?

Reply:
Yes!